src/Language/NodePrinter.php

```diff
@@ -30,11 +32,11 @@
 use Digia\GraphQL\Language\Node\VariableNode;
 use function Digia\GraphQL\Util\toString;
 
-class Printer implements PrinterInterface
+class NodePrinter implements NodePrinterInterface
```

Comment:
After renaming this class I realized that we should probably move both this and the `NodeBuilder` into the `Node` namespace as it uses quite a few classes from that namespace and the class name also starts with the word `Node`.

Reply:
Agreed.